build.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
     api group: 'com.azure', name: 'azure-data-tables'
     api group: 'com.microsoft.azure', name: 'azure-batch', version: '10.1.0'
     api group: 'com.microsoft.azure', name: 'azure-storage', version: '8.6.6'
+    api group: 'com.azure', name: 'azure-messaging-eventhubs-checkpointstore-blob', version: '1.18.1'
+    api group: 'com.azure', name: 'azure-messaging-eventhubs', version: '5.17.1'
```

Comment:
The version should be part of the BOM, if so, they must be removed

Reply:
The versions was removed and bom version updated.